lang/src/org/partiql/lang/syntax/LexerConstants.kt

```diff
@@ -441,9 +441,19 @@ internal val DATE_TIME_PART_KEYWORDS: Set<String> = DateTimePart.values()
     listOf("not", "in") to ("not_in" to OPERATOR),
     listOf("is", "not") to ("is_not" to OPERATOR),
     listOf("not", "between") to ("not_between" to OPERATOR),
+
+    listOf("union", "all") to ("union_all" to OPERATOR),
```

Comment:
(not completely relevant to this PR's changes but may be relevant to future set operator PRs): I know that there's an implicit `DISTINCT` when no set quantifier is provided for the set operators, but I think the lexer isn't properly handling the case when the `DISTINCT` set quantifier is defined.

E.g:
```
PartiQL> A UNION DISTINCT B;
org.partiql.lang.syntax.ParserException: Unexpected keyword
        Parser Error: at line 1, column 9: unexpected keyword found, KEYWORD : distinct
```

Since it's part of the [RFC](https://github.com/partiql/partiql-docs/blob/main/RFCs/0007-rfc-bag-operators.md#definition), we may need to handle the explicit `DISTINCT` cases as well.

Reply:
I'll fix the DISTINCT in the next revision